rest/blip_api_test.go

```diff
@@ -2485,7 +2485,258 @@ func TestBlipDeltaSyncPullRevCache(t *testing.T) {
 
 	assert.Equal(t, deltaCacheHits+1, updatedDeltaCacheHits)
 	assert.Equal(t, deltaCacheMisses, updatedDeltaCacheMisses)
+}
```

Comment:
As discussed, the changes to GetDelta can probably be tested more easily by testing GetDelta directly, and verifying it's returning the expected result (or error in the missing case).

Reply:
Added two tests TestDeltaSyncWhenFromRevIsChannelRemoval and TestDeltaSyncWhenToRevIsChannelRemoval to check the delta sync behavior.